src/test/java/uk/gov/register/functional/app/RegisterRule.java

```diff
@@ -81,6 +81,10 @@ public Response getRequest(String path) {
         return target().path(path).request().get();
     }
 
+    public Response deleteRequest(String path) {
```

Comment:
does it make sense to make this specific to the only DELETE endpoint that we have?